### DISCUSSION TITLE
App on iPad Mini is weirdly slow and non-functional.

### DISCUSSION THREAD

**Discussion by DoubleF3lix** [reflex-dev/reflex#341] [Q&A]
I tried out a sample application today which just has some text, and a button that should make another text field appear when it's clicked. It works fine on my PC, but when I attempt to use the site on my iPad Mini (iOS 12, not sure what version the hardware is though), it takes forever to load, and when it does, the button doesn't function as I expect it would. 

Is it just that pynecone doesn't work on older builds of Safari, or is it possible that I'm doing something else wrong?
Here's my code:
```py
class State(pc.State):
    """The app state."""
    good = False

    def push_btn(self):
        self.good = True


def index():
    return pc.center(
        pc.vstack(        
            pc.hstack(
                pc.text("Click here to "),
                pc.button("show text", on_click=[State.push_btn]),
            ),
            pc.cond(
                State.good,
                pc.text("Good job!"),
            )
        )
    )


# Add state and page to the app.
app = pc.App(state=State)
app.add_page(index, title="Site")
app.compile()
```

**Comment by Alek99** [comment]
Are you running this app in prod or dev mode?

**Reply by DoubleF3lix** [reply]
Dev mode

**Reply by Alek99** [reply]
Dev mode is usually slower try running in prod mode `pc run --env prod` this will run a productionized build which should be faster but will not allow for the hot reload feature. In general anytime your app is used on other device I would suggest running in prod mode, only use dev mode for developing and testing locally.

**Reply by DoubleF3lix** [reply]
I've done this, and now when I run it locally, the button pressing action does not work, however it does when running in dev mode. However, loading on iPad initially is much faster now :D

**Reply by Alek99** [reply]
Ok thanks for the feedback I'll try and debug this asap prod mode seems to work for me. Our main website/docs is made with Pynecone and we run it on prod mode. If you have time feel free to join the discord and I can try and help further, I'll try and get to this over the weekend.
